src/config/kyuubi.py

```diff
@@ -8,18 +8,25 @@
 from typing import Optional
 
 from constants import AUTHENTICATION_TABLE_NAME, HA_ZNODE_NAME
-from core.domain import DatabaseConnectionInfo, ZookeeperInfo
+from core.domain import DatabaseConnectionInfo, TLSInfo, ZookeeperInfo
+from core.workload import KyuubiWorkloadBase
 from utils.logging import WithLogging
 
 
 class KyuubiConfig(WithLogging):
     """Kyuubi Configurations."""
 
     def __init__(
-        self, db_info: Optional[DatabaseConnectionInfo], zookeeper_info: Optional[ZookeeperInfo]
+        self,
+        db_info: Optional[DatabaseConnectionInfo],
+        zookeeper_info: Optional[ZookeeperInfo],
+        tls: Optional[TLSInfo],
```

Comment:
**nitpick**: Out of scope, but this type annotation made me realize that we are also using Python 3.8 syntax on this charm though we have 3.10 at runtime. Moving on to 3.10 would mean that we would use `TLSInfo | None`

Reply:
yes, that's a good point. Can we put a ticket in the backlog for this and link it to this comment?